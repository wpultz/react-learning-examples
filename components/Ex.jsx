import React from 'react'

const Ex = () => <div className="gamepiece">X</div>

export default Ex
