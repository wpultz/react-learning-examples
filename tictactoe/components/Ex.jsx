import React from 'react'

const Ex = () => <div className="gamepiece"><div className="gamepiece-content"><img className="gamepiece-x" src="/assets/upthere.png" /></div></div>

export default Ex
