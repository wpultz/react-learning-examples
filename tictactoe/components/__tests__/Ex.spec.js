import React from 'react'
import renderer from 'react-test-renderer'

import Ex from '../Ex'

it('renders an X', () => {
  const tree = renderer.create(<Ex />).toJSON()

  expect(tree).toMatchSnapshot()
})