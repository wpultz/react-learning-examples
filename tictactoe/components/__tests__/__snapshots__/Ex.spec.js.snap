// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an X 1`] = `
<div
  className="gamepiece"
>
  <span
    className="gamepiece-content"
  >
    X
  </span>
</div>
`;
